### ISSUE TITLE
TypeError: Cannot read property '0' of null when uploading an image using the webcam

### ISSUE THREAD

**Issue by ericvillemure** [filestack/filestack-js#20]
Hi guys I am calling filestackClient.pick with the filestack-js V3 and getting this error when taking a photo with my webcam and then clicking Next:

TypeError: Cannot read property '0' of null

Seems to inside this function:
```javascriptvar 
extractExtension = function extractExtension(filename) {
   return /\.\w+$/.exec(filename)[0]
};
```
The filename is "webcam image" and it's causing the crash

If you need the exact options passed to the pick function let me know I will find that for you but I think it should be easy to repro.

Windows 10
Chrome Version 58.0.3029.110 (64-bit)

**Comment by cogKnitter** [comment]
Thanks for the information @ericvillemure.
We will replicate and find a fix.

**Comment by velveteer** [comment]
This will be fixed in 0.7.0.
